frontend_app_agenda/src/container/Agenda.jsx

```diff
@@ -32,50 +35,89 @@ class Agenda extends React.Component {
+    ])
 
-  customEventReducer = ({ detail: { type, data } }) => {
-    const { state, props } = this
+    props.registerLiveMessageHandlerList([
+      { entityType: TLM_ET.USER, coreEntityType: TLM_CET.MODIFIED, handler: this.handleUserModified },
+      { entityType: TLM_ET.SHAREDSPACE, coreEntityType: TLM_CET.MODIFIED, handler: this.handleSharedspaceModified }
+    ])
+  }
 
-    switch (type) {
-      case CUSTOM_EVENT.SHOW_APP(state.config.slug):
-        console.log('%c<Agenda> Custom event', 'color: #28a745', type, data)
-        if (data.config.appConfig.workspaceId !== state.config.appConfig.workspaceId) {
-          this.setState({ config: data.config })
-        }
-        break
-      case CUSTOM_EVENT.ALL_APP_CHANGE_LANGUAGE:
-        console.log('%c<Agenda> Custom event', 'color: #28a745', type, data)
-        this.setState(prev => ({
-          loggedUser: {
-            ...prev.loggedUser,
-            lang: data
-          }
-        }))
-        i18n.changeLanguage(data)
-        this.buildBreadcrumbs()
-        this.setHeadTitle(state.config.appConfig.workspaceId !== null
-          ? `${props.t('Agenda')} · ${state.content.workspaceLabel}`
-          : props.t('My agendas')
-        )
-        this.agendaIframe.contentWindow.location.reload()
-        break
-      default:
-        break
+  // Custom Event Handlers
+  handleShowApp = data => {
+    console.log('%c<Agenda> Custom event', 'color: #28a745', CUSTOM_EVENT.SHOW_APP(this.state.config.slug), data)
+    if (data.config.appConfig.workspaceId !== this.state.config.appConfig.workspaceId) {
+      this.setState({ config: data.config })
     }
   }
 
+  handleAllAppChangeLanguage = data => {
+    const { props, state } = this
+    console.log('%c<Agenda> Custom event', 'color: #28a745', CUSTOM_EVENT.ALL_APP_CHANGE_LANGUAGE, data)
+
+    this.setState(prev => ({
+      loggedUser: {
+        ...prev.loggedUser,
+        lang: data
+      }
+    }))
+    i18n.changeLanguage(data)
+    this.buildBreadcrumbs()
+    this.setHeadTitle(state.config.appConfig.workspaceId !== null
+      ? `${props.t('Agenda')} · ${state.content.workspaceLabel}`
+      : props.t('My agendas')
+    )
+    this.agendaIframe.contentWindow.location.reload()
+  }
+
+  // TLM Handlers
+  handleUserModified = data => {
+    const { state } = this
+    if (state.loggedUser.userId !== data.user.user_id) return
+
+    this.setState(prev => ({
+      loggedUser: {
+        ...prev.loggedUser,
+        authType: data.user.auth_type,
+        avatarUrl: data.user.avatar_url,
+        email: data.user.email,
+        isActive: data.user.is_active,
+        profile: data.user.profile,
+        publicName: data.user.public_name,
+        timezone: data.user.timezone,
+        username: data.user.username
+      },
+      // INFO - GB - 2020-06-18 - Just show the message in "My agendas" page and if it's not the language that changes (handled by custom event)
```

Comment:
add some more info about what is "the message" and what means `state.userWorkspaceList.length !== 1`